examples/data-science/pandas/CDC_NDI_Mortality_Syft_Data_Scientist.ipynb

```diff
@@ -0,0 +1,292 @@
+{
```

Comment:
<p>TYPO: <code>DIABETES</code>  -&gt; <code>HEART</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/5152/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>I don't think this is a typo, unless im mis-understanding you. This is the DIABETES column for both cancer and heart deaths.</p>